code/VersionedGridFieldDetailForm.php

```diff
@@ -76,8 +105,14 @@ public function canDelete() {
 		return $this->record->canDelete();
 	}
 
-	public function canPreview() {
```

Comment:
Personally I think it's always good practice to explicitly mark function visibility. I realise this is the same as "public function", but I think there are some developers who need to be reminded.